### PR TITLE
Switch to NSNumber* accountNo

### PR DESCRIPTION
**Pull request by FriedrichAltheide**

### CODE REVIEW

File: Monal/Classes/MLLogInViewController.m [hunk 1]
```diff
@@ -214,7 +215,14 @@ -(void) updateBundleFetchStatus:(NSNotification*) notification
 
 -(void) omemoBundleFetchFinished:(NSNotification*) notification
 {
-    if(notification == nil || [notification.userInfo[@"accountNo"] isEqualToString:self.accountNo])
+    BOOL handleNotification;
+    if(notification == nil)
+        handleNotification = YES;
+    else{
```
Comment (FriedrichAltheide) [inline Monal/Classes/MLLogInViewController.m]:
```suggestion
    else
    {
```

**Comment by FriedrichAltheide** [comment]
@tmolitor-stud-tu ping

File: Monal/Classes/DataLayer.m [hunk 2]
```diff
@@ -363,9 +363,9 @@ -(void) persistState:(NSDictionary*) state forAccount:(NSString*) accountNo
 
 #pragma mark contact Commands
 
--(BOOL) addContact:(NSString*) contact forAccount:(NSString*) accountNo nickname:(NSString*) nickName andMucNick:(NSString* _Nullable) mucNick
+-(BOOL) addContact:(NSString*) contact forAccount:(NSNumber*) accountNo nickname:(NSString*) nickName andMucNick:(NSString* _Nullable) mucNick
 {
-    if(!accountNo || !contact)
+    if(accountNo == nil || !contact)
```
Comment (tmolitor-stud-tu) [inline Monal/Classes/DataLayer.m:368]: this is not necessary

**Review by tmolitor-stud-tu — APPROVED** [review]
+1

**Pull request merged**